### PR TITLE
docs(Advanced guides): React without ES6

### PR DESCRIPTION
**Pull request by henry61024**
Translate Advanced guides - `React without ES6`

### CODE REVIEW

**Comment by henry61024** [comment]
I would like to discuss about the translate of this articles title `React Without ES6`.

I personally think that `不使用 ES6 開發 React` is more fluent than the original translate `沒有 ES6 的 React`, how do you think of that?

However, the slight problem about choosing `不使用 ES6 開發 React` is that we'll need to unify the translation of `React Without JSX` as well.

File: content/docs/react-without-es6.md [hunk 1]
```diff
@@ -26,11 +26,11 @@ var Greeting = createReactClass({
 });
 ```
 
-The API of ES6 classes is similar to `createReactClass()` with a few exceptions.
+ES6 class 與 `createReactClass()` 的 API 十分相似，只有以下幾個不同處。
 
-## Declaring Default Props {#declaring-default-props}
+## 宣告預設 props {#declaring-default-props}
```
Comment (neighborhood999) [inline content/docs/react-without-es6.md:31]:
```suggestion
## 宣告預設 Props {#declaring-default-props}
```

**Review by neighborhood999 — COMMENTED** [review]
Nice!

File: content/docs/react-without-es6.md (hunk 1 above)
Comment (henry61024) [reply content/docs/react-without-es6.md]: Fixed in 9288dfc

**Comment by henry61024** [comment]
> I would like to discuss about the translation of this articles' title: `React Without ES6`.
> 
> I personally think that `不使用 ES6 開發 React` is more intuitive than the original translation `沒有 ES6 的 React`, what is your thought on this?
> 
> However, the slight downside about choosing `不使用 ES6 開發 React` is that we may need to unify the translation of `React Without JSX` as well.

Hi, any thoughts on this?

**Comment by neighborhood999** [comment]
> I would like to discuss about the translation of this articles' title: `React Without ES6`.
> 
> I personally think that `不使用 ES6 開發 React` is more intuitive than the original translation `沒有 ES6 的 React`, what is your thought on this?
> 
> However, the slight downside about choosing `不使用 ES6 開發 React` is that we may need to unify the translation of `React Without JSX` as well.

I think your opinion is great, can you help to update `React Without JSX` docs?

**Comment by henry61024** [comment]
Of course, I'm glad to help!

**Comment by henry61024** [comment]
> > I would like to discuss about the translation of this articles' title: `React Without ES6`.
> > I personally think that `不使用 ES6 開發 React` is more intuitive than the original translation `沒有 ES6 的 React`, what is your thought on this?
> > However, the slight downside about choosing `不使用 ES6 開發 React` is that we may need to unify the translation of `React Without JSX` as well.
> 
> I think your opinion is great, can you help to update `React Without JSX` docs in this PR?

Updated in 78ee1a2

**Review by neighborhood999 — APPROVED** [review]

**Pull request merged**